ex1/ex1/Exercises/Student.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ex1.Exercises
+{
+    public class Student
+    {
+        private string Name { get; set; }
```

Comment:
Similar thoughts here as in **Mark** class above. If we need to store some data privately we usually add private fields. In order to access these private data, we can add corresponding public properties. So, either add private fields and set them using constructor or just add public properties. I suggest you read in Troelson about the purpose of properties